stable/rabbitmq/values.yaml

```diff
@@ -54,6 +54,12 @@ rabbitmqVhost: /
 ##
 rabbitmqManagerPort: 15672
 
+## RabbitMQ Disk free limit
+## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
+## ref: https://www.rabbitmq.com/disk-alarms.html
+##
+rabbitmqDiskFreeLimit: '"8GiB"'
```

Comment:
Why is this quoted twice?

Reply:
Because you have to pass a quoted string to rabbitmq (https://www.rabbitmq.com/disk-alarms.html)